apps/api/v2/src/ee/bookings/2024-04-15/controllers/bookings.controller.ts

```diff
@@ -49,8 +49,12 @@ import {
   getBookingInfo,
   handleCancelBooking,
   getBookingForReschedule,
-} from "@calcom/platform-libraries";
-import { GetBookingsInput, CancelBookingInput, Status } from "@calcom/platform-types";
+} from "@calcom/platform-libraries-1.2.3";
```

Comment:
we should not updated platform-libraries anymore in this controller right ?

Reply:
Well spotted!
1. [handleNewRecurringBooking.ts](https://github.com/calcom/cal.com/pull/16200/files#diff-7beacbbd958787bafebd00426229d723b75d465b0f4f15d5d1682f0ccea18e91) has been updated to receive `noEmail` on the request.
2. In this legacy bookings controller [here](https://github.com/calcom/cal.com/pull/16200/files#diff-48f6f92aff2c62446cca48a7dc3139b726861c3a89a510a7ffa1eeda3c4e7f99R376) I add the `noEmail` to the request.

Shortly, `handleNewRecurringBooking` has been updated and legacy controller needs it because `noEmail` was ignored thus far.